tools/scxtop/src/perfetto_trace.rs

```diff
@@ -248,6 +249,12 @@ impl PerfettoTraceManager {
         }
     }
 
+    fn get_comm(pid: i32) -> Result<String> {
```

Comment:
nit: there's already a [helper function](https://github.com/sched-ext/scx/blob/fd88ee611130433371c37474f449985abb975d35/tools/scxtop/src/util.rs#L11)

Reply:
Thanks @hodgesds for pointing this out! I'll try and refactor some time to use that interface.